frontend/src/services/event.ts

```diff
@@ -51,3 +51,17 @@ export const updateEvent = (event: Event) =>
 
 export const deleteEvent = (id: string): Promise<Response> =>
   deleteData(`events/${id}`, withAuthenticationHeader());
+
+export const checkinDepartmentToEvent = (
+  event: Event,
+  departmentId: string
+) => {
+  return postData<AttendeeResponse[]>(
+    `events/add-department/${event.code}/${departmentId}`,
+    withAuthenticationHeader(),
+    {}
+  );
+};
+
+export const getEventByType = (type: EventType) =>
+  getData<Event>(`events/by-type/${type}`, withAuthenticationHeader());
```

Comment:
Vielleicht zu naiv gedacht, aber warum packst du das `withAuthentitcationHeader` nicht gleich in die einzelnen http-Methoden?